src/app/feed/feed-post-icon-row/feed-post-icon-row.component.ts

```diff
@@ -41,6 +41,12 @@ export class FeedPostIconRowComponent {
   // Threshold above which user must confirm before sending diamonds
   static DiamondWarningThreshold = 3;
 
+  // Boolean for animation on whether a heart is clicked or not
+  clicklike = false;
```

Comment:
Can we rename this to `animateLike` and same thing for diamond? Looks good to me otherwise

Reply:
Will do!